File: Framework-Samples/NHibernate/CSharp/Sample.Data/ManagerObjects/CartManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using Sample.Data.Generated.BusinessObjects;
using Sample.Data.Generated.Base;

namespace Sample.Data.Generated.ManagerObjects
{
    public partial interface ICartManager : IManagerBase<Cart, int>
    {
	}
	
	partial class CartManager : ManagerBase<Cart, int>, ICartManager
    {
	}
}

File: Framework-Samples/NHibernate/VisualBasic/Sample.Data/ManagerObjects/CartManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports Sample.Data.Generated.BusinessObjects
Imports Sample.Data.Generated.Base

Namespace Sample.Data.Generated.ManagerObjects
	'Public Partial Interface ICartManager
	'	Inherits IManagerBase(Of Cart, System.Int32)
	'End Interface

	Partial Class CartManager
		Inherits ManagerBase(Of Cart, System.Int32)
		Implements ICartManager
	End Class
End Namespace
